src/LocalMeanHeuristics.cpp

```diff
@@ -0,0 +1,16 @@
+#include "LocalMeanHeuristics.h"
+
+bool LocalMeanHeuristics::belongToCluster(libMesh::Elem *elem,libMesh::Elem *neighbor_elem){
+
+    double local_score =0;
+    unsigned int number_of_active_neighbor =0;
+    for (unsigned int side =0;side< neighbor_elem->n_sides();side++){
+        const libMesh::Elem *current_elem = neighbor_elem->neighbor_ptr(side);
+        if (current_elem){
+            local_score+=getElementDataFromMesh(current_elem);
```

Comment:
Might want to apply clang-format for style
```suggestion
            local_score += getElementDataFromMesh(current_elem);
```